packages/react/src/utils/useAnchorPositioning.ts

```diff
@@ -118,12 +122,29 @@ export function useAnchorPositioning(
   // presence.
   const arrowRef = React.useRef<Element | null>(null);
 
+  // Keep these reactive if they're not functions
+  const sideOffsetDep = typeof sideOffset === 'number' ? side : 0;
+  const alignOffsetDep = typeof alignOffset === 'number' ? align : 0;
```

Comment:
Positioning reactivity is lost when they specify a function; we can't tell if they're going to `useCallback` or not, usually the function gets inlined like:

```jsx
<Popover.Positioner sideOffset={sizes => sizes.anchor.height / 2}>
```